04/app.js

```diff
@@ -1,89 +1,92 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
+import React from 'react'
+import { createRoot } from 'react-dom/client'
 
-const root = createRoot(document.querySelector('#root'));
+const root = createRoot(document.querySelector('#root'))
 
 class App extends React.Component {
-    state = { 
-        firstName: '',
-        lastName: '',
-        searchQuery: '',
-        users: ['Jan Kowalski', 'Michał Nowak'],
-    }
+	state = {
+		firstName: '',
+		lastName: '',
+		searchQuery: '',
+		users: ['Jan Kowalski', 'Michał Nowak'],
+	}
 
-    renderUsersList() {
-        const {users} = this.state;
-        return users.map(name => {
-            return (
-                <li onClick={ this.clickHandler }>
-                    { name }
-                </li>
-            );
-        });
-    }
+	renderUsersList() {
+		const { users, searchQuery, filteredUsers } = this.state
+		if (searchQuery) {
+			this.filterUsers()
```

Comment:
Gdzie `return`? :P Warto sprawdzić 3 razy czy działa zanim oddamy - na rekrutacji to niepotrzebny OGROMNY błąd, który może nas skreślić :(